src/interfaces/methods/network-v2.ts

```diff
@@ -48,4 +48,6 @@ export interface Network_v2Methods {
   disputeBountyProposal(bountyId: number, proposalId: number): ContractSendMethod;
   refuseBountyProposal(bountyId: number, proposalId: number): ContractSendMethod;
   closeBounty(id: number, proposalId: number): ContractSendMethod;
+  isBountyInDraft(bountyId: number): Promise<boolean>;
```

Comment:
this file relates to the methods _inside the contract_ and not the ones _on the class_, since `npm run docs` is smart enough, you don't need to add their definition here lest they appear on the `.methods.` object (which they wont)